en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -80,142 +76,132 @@ For example, see the video ID circled in red in the illustration below: `q2l-8-r
+{% include figure.html filename="en-or-text-mining-youtube-comments-03.png" alt="Screenshot image of YouTube Data Tools software displaying the data entry fields for video search, as well as other download options" caption="Figure 3. Screenshot of YouTube Data Tools webpage for downloading video comments" %}
+
+Enter the first video ID in the _Video id_ field. You can only download comment data pertaining to one video ID at a time. We recommend leaving the _Limit to_ field blank. Next, you must make three choices about your data's output format: 
 
-Thirdly, select .csv in the _File Format_ parameter.
+1) _HTML output_: check the box if you’d like to generate HTML tables of your results in addition to file exports. This parameter gives you a preview of what you are downloading, which can be useful to monitor the downloading process. 
+2) _Pseudonymize_: check the box to add irreversible hashes to the comment usernames and ID numbers, rendering them anonymous for the purposes of privacy. We mentioned this in the [Ethical Considerations section](#Ethical-Considerations-for-Social-Media-Analysis).
+3) _File Format_: select CSV.
 
 Finally, press _Submit_. 
 
 Repeat this process for each video. More details on this download process can be found in Bernhard Reider's [instructional video](https://www.youtube.com/watch?v=EnTy_pbkCfM).
 
 ### YouTube Data Tools Output
 
-Downloading data from the "Video Comments" tab on YouTube Data Tools will generate four files. Each automatically generated filename contains key pieces of information: the video ID, the download date, and a description of the file content. 
-
-{% include figure.html filename="en-or-text-mining-youtube-comments-04.png" alt="Screenshot of the four files generated by YouTube Data Tools, with highlights on the two files used in this lesson's code, entitled 'authors' and 'comments'." caption="Figure 4. File structure of YouTube Data Tools output" %}
+Downloading data from the **Video Comments** tab on YouTube Data Tools will generate four files. Each automatically generated filename contains key pieces of information: the video ID, the download date, and a description of the file content. In this lesson, you'll only need the files containing `comments` and `basicinfo`.
 
-The video ID and download date in the filename are particularly important to retain because new YouTube comments may continue to be uploaded after your data was collected. The video ID follows immediately after the phrase "videoinfo." The download date (the date the files were generated and downloaded) follows the video ID. 
+{% include figure.html filename="en-or-text-mining-youtube-comments-04.png" alt="Screenshot of the four files generated by YouTube Data Tools, with highlights on the two files used in this lesson's code, entitled 'comments' and 'basic info'." caption="Figure 4. File structure of YouTube Data Tools output" %}

```

Comment:
I noted your comment that highlighting certain files without an explanation would be confusing. Actually, I realised we were highlighting the wrong files (authors instead of basicinfo), so I've replaced the image again. I think it makes sense to highlight those two files if we add the clarifications: 'In this lesson, you'll only need the files containing `comments` and `basicinfo`.(line 127) and '(As mentioned above, you won't need all four files for this lesson, but `authors` and `commentnetwork` contain additional information that could be useful for potential further research.)' (line 140). 

What do you think?

Reply:
sounds great!